packages/react-provider/src/transports/types.ts

```diff
@@ -33,16 +34,32 @@ export interface ICourierMessage {
     trackingUrl?: string;
     clickAction?: string;
   };
+  error?: string;
```

Comment:
are these optional to support backward compatibility?

Reply:
error is optional because it doesn't exist if there is no error